__DOC_EXAMPLES__/gettingStarted/terminology.md

```diff
@@ -0,0 +1,67 @@
+<h1>SDK Terminology</h1>
+
+<h4>The best way to explain our terminology is through an example</h4>
+
+
+```javascript
+
+  import Cloudinary, {TransformableImage} from '@cloudinary/base';
+
+  // Create your instance
+  const cld = new Cloudinary(/* ... config */);
```

Comment:
I think it would be useful to show the whole config here, as you have in other examples.